docs/topics/db/models.txt

```diff
@@ -910,14 +910,14 @@ example::
 
     class Blog(models.Model):
         name = models.CharField(max_length=100)
-        slug = models.TextField()
+        slug = models.SlugField(max_length=100)
```

Comment:
While I do agree that this is more correct I believe we should keep the patch focused on the `update_fields` part. Not a huge deal either way, and something mergers will be able to figure out.

Reply:
Good point, I will backport this so it's nice to keep it focused on the issue :+1: